proposals/vertical-rhythm.md

```diff
@@ -0,0 +1,97 @@
+# Vertical rhythm
+
+## Current implementations
+
+We have two main ways to control vertical rhythm: our `spacing` utility and the `<Section />` component.
+
+The `spacing` utility takes one of five string values (`xl`, `lg`, `md`, `sm`, `none`) or a default and returns css with `margin-bottom`.
+
+For example, calling `spacing('md')` returns the following CSS: `margin-bottom: 20px;`
+
+The `<Section />` component is simply a `div` with two variants: `section` and `semi-section`.
+
+The former gives the `div` a `margin-bottom` of `100px` and the latter of `50px` (these are for desktop, and values vary on other devices).
+
+## Problems
+
+### `margin-top` on the first section
+
+If a layout consists of sections with 100px of space above and below, using tools that only tackle `margin-bottom` we are unable to provide the spacing at the top of the layout.
+
+### `spacing` is a confusing name
+
+Our utility currently only adds `margin-bottom`. Spacing could imply `margin` **_or_** `padding` in **_any_** direction.
+
+### absolute values for `margin`
+
+While we generally limit the use of absolute values for `margin-bottom` in `festicket-app` by using the `spacing` utility, the many exceptions and **all** values for `margin-top` remain absolute and therefore difficult to maintain.
+
+## Proposal
+
+We maintain the `spacing` utility and `<Section />` component, but with modifications.
+
+The `spacing` module will be renamed and will allow an optional second argument to specify the direction of the margin.
+
+```js
+function rhythm(size, direction = 'bottom') {
```

Comment:
I think this name is worse than spacing tbh. One of the problems states is that spacing is a confusing name because it could imply space in any direction, you've added that functionality here so surely the name makes sense now?

Reply:
or it can by renamed to [vSpacing | vspacing] - to make it more specific

Reply:
Its supposed to map to vertical rhythm rather than spacing. Maybe we should just name it that?

Reply:
gonna go with `verticalMargin` so there can be no confusion